tests/nfv/trex_installation.pm

```diff
@@ -23,24 +19,20 @@ use mmapi;
 use serial_terminal 'select_virtio_console';
 
 sub run {
-    my $trex_repo = "https://github.com/cisco-system-traffic-generator/trex-core.git";
-    my $trex_dest = "/tmp/trex-core";
+    my $trex_version = get_var('TG_VERSION');
```

Comment:
What's about using get_required_var here to fail for a clear reason when variable is not set? WDYT?

Reply:
it makes sense as it is required. It's updated now. Thanks.